Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 AS builder
```

Comment:
Current openshift is already on 1.16, is it possible to make this Dockerfile 1.16 as well?

Reply:
Yes, let me try that.